protocol-configuration/collateralizationOracle.ts

```diff
@@ -3,7 +3,6 @@
 // Values are arrays of contracts that hold that token
 
 export const CollateralizationOracleConfig = {
-  fei: ['rariTimelockFeiOldLens'],
```

Comment:
the fei in this contract still needs to be reported as protocol owned until its burned

Reply:
Re-added